Fix linux support with disk encryption (#5909)
cloudflare/cloudflare-docs: content/cloudflare-one/identity/devices/warp-client-checks/disk-encryption.md

1. **Protection Status** must be set to **On**.

All disks on the system must be encrypted for the posture check to pass.

### On Linux

> Disk encryption is not supported on Linux at the moment

Reviewer: ```suggestion
Disk encryption checks are not currently supported on Linux.
```
